### ISSUE TITLE
package-lock.json and other build instructions

### ISSUE THREAD

**Issue by zbeekman** [j2kun/harmonic-phase-space#2]
Hi,

This looks pretty interesting. I first tried to see if you deployed this via GH pages or similar, but it looks like the answer is "no." (I tried visiting http://j2kun.github.io/harmonic-phase-space .)

Next, I cloned and tried to build it locally on macOS Mojave with the latest brewed node.js but I ran into the following error:

```
node_modules/.bin/gulp build
fs.js:27
const { Math, Object } = primordials;
                         ^

ReferenceError: primordials is not defined
    at fs.js:27:26
    at req_ (/Users/ibeekman/Sandbox/harmonic-phase-space/node_modules/natives/index.js:143:24)
    at Object.req [as require] (/Users/ibeekman/Sandbox/harmonic-phase-space/node_modules/natives/index.js:55:10)
    at Object.<anonymous> (/Users/ibeekman/Sandbox/harmonic-phase-space/node_modules/vinyl-fs/node_modules/graceful-fs/fs.js:1:37)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)
```

I suspect that adding a `package-lock.json` file may resolve my issue, or building with a different version of node.

* What version of node are you using?
* Could you add a `package-lock.json` file?
* Would it be possible to add the required build commands as aliases to `package.json`?
* Is this already deployed somewhere? If not could it be?

Thanks so much!

**Comment by j2kun** [comment]
Hey there! You stumbled upon this nascent project mere hours after I pushed it to Github for the first time 😅

I ran into the same error just now with Node 12, found out I had built it with Node 10.8.0. I just confirmed that the build script works using nvm

```
nvm install 10.8.0
nvm use 10.8.0
gulp build
open -a Google\ Chrome index.html
```

Of course, it doesn't do much yet. You joined the party a few days too early :) I'll add a proper readme when it's ready.

I also added a yarn.lock file. Not sure if that is equivalent to package-lock.json
